src/enums/commands.enum.ts

```diff
@@ -0,0 +1,3 @@
+export enum Commands {
```

Comment:
Sometimes you add the suffix enum on your enums.
Sometimes not.
Choose please.

Reply:
I will add the enum suffix... ><

Is it possible to check this with a linter? It could be greatly nice xD

Reply:
This is a good idea.
This is not the case on my project but I will open an issue to find more about this possibility.
I will handle this after I finished my current feature (probably tomorrow).

Reply:
https://github.com/Sonia-corporation/il-est-midi-discord/issues/853